_includes/head

```diff
@@ -26,21 +26,21 @@
 
 	<!-- Favicon/touch icons -->
 	<link rel="shortcut icon" href="/favicon.ico"> <!-- must remain in root directory -->
-	<link rel="icon" type="image/png" href="{{site.url}}/assets/img/favicon-16x16.png" sizes="16x16">
-	<link rel="icon" type="image/png" href="{{site.url}}/assets/img/favicon-32x32.png" sizes="32x32">
-	<link rel="icon" type="image/png" href="{{site.url}}/assets/img/favicon-96x96.png" sizes="96x96">
-	<link rel="apple-touch-icon" sizes="57x57" href="{{site.url}}/assets/img/apple-touch-icon-57x57.png">
-	<link rel="apple-touch-icon" sizes="60x60" href="{{site.url}}/assets/img/apple-touch-icon-60x60.png">
-	<link rel="apple-touch-icon" sizes="72x72" href="{{site.url}}/assets/img/apple-touch-icon-72x72.png">
-	<link rel="apple-touch-icon" sizes="76x76" href="{{site.url}}/assets/img/apple-touch-icon-76x76.png">
-	<link rel="apple-touch-icon" sizes="114x114" href="{{site.url}}/assets/img/apple-touch-icon-114x114.png">
-	<link rel="apple-touch-icon" sizes="120x120" href="{{site.url}}/assets/img/apple-touch-icon-120x120.png">
-	<link rel="apple-touch-icon" sizes="144x144" href="{{site.url}}/assets/img/apple-touch-icon-144x144.png">
-	<link rel="apple-touch-icon" sizes="152x152" href="{{site.url}}/assets/img/apple-touch-icon-152x152.png">
-	<link rel="apple-touch-icon" sizes="180x180" href="{{site.url}}/assets/img/apple-touch-icon-180x180.png">
-	<link rel="icon" type="image/png" href="{{site.url}}/assets/img/android-chrome-192x192.png" sizes="192x192">
-	<link rel="manifest" href="{{site.url}}/assets/img/manifest.json">
-	<link rel="mask-icon" href="{{site.url}}/assets/img/safari-pinned-tab.svg" color="#293c92">
+	<link rel="icon" type="image/png" href="/assets/img/favicon-16x16.png" sizes="16x16">
```

Comment:
I'm not sure what `site.baseurl` is for (is it just to avoid having to use `root`?), but we probably want to either use it in places like this or remove it everywhere for consistency.